model/vm.rb

```diff
@@ -15,6 +15,7 @@ class Vm < Sequel::Model
   one_to_many :pci_devices, key: :vm_id, class: :PciDevice
   one_through_one :load_balancer, left_key: :vm_id, right_key: :load_balancer_id, join_table: :load_balancers_vms
   one_to_one :load_balancers_vms, key: :vm_id, class: LoadBalancersVms
+  one_to_one :inference_endpoint_replica, key: :vm_id
```

Comment:
Should VM, which is a more general concept refer to inference_endpoint_replica. I expect many VMs to not have inference_endpoint_replica.

Instead, maybe it would make more sense to pass readonly and image name as a fields in storage devices? (I'm not sure what would make sense, but overall I don't think lower level services like VM should refer to higher level ones if possible)

Reply:
you are right. got rid of this. I now pass two volumes when creating vms for inference replicas: the boot volume and a new "read_only" volume.